code/drasil-example/Drasil/Projectile/ProblemDescription.txt

```diff
@@ -0,0 +1,44 @@
+﻿Physical System Description
+
+Elements:
+	1) The object
+	2) The ground
+
+Assumptions
+1) The projectile motion is in 2D.
+2) The object is a point mass.
+3) The object's path starts and ends at ground level.
+4) The effects of drag, wind, and the Earth's curvature are ignored.
+5) The flight is free; there are no collisions during the trajectory of the object.
+6) The positive x-direction is the direction from start to the end of the object's path.
+7) The positive y-direction is the direction away from the ground.
+
+Goal Statments
+1) Check if the final position from user input is reached by the object with initial conditions.
+2) If the final position is not reached, provide the offset and if the object went long or short.
+
+Theoretical Models
+1) v = ds/dt
+2) a = dv/dt
+
+General Definitions
+1) v_f = v_i + a*t
+2) d = v_i_x*t + a_x*(t^2)/2
+3) d = 2*(v_i^2)*sin(θ)*cos(θ)/(|a|) (from A4, GD2)
+	- v at max height = 0
+	- t at max height = t_total/2
+
+Data Definitions
+1) v_x = v*cos(θ)
+2) v_y = v*sin(θ)
+
+Instance Models
```

Comment:
I think I'd prefer these instance models as requirements.  I think the instance model is simply solving for the distance travelled by the projectile.  In the current set-up for Drasil though, it might be easier to have instance models.

Reply:
So should I leave it for now?

Reply:
I'd like to see what @bmaclach thinks.  It depends on the code generator, and I don't know if the code generator starts from the instance models or the requirements.  I know that it should eventually start from the requirement, which reference the instance models, but I'm not sure if that is how things are implemented.

Reply:
The code generator uses the instance models to calculate the outputs, and then just returns the values of those outputs. So if we don't include these instance models, the generated code would return the displacement and nothing else. I'm pretty sure the code generator ignores requirements altogether.

Reply:
I think in the long term we want the requirements to be more meaningful, but for the current situation, we can use the instance models.  We should still write requirements though.